pkg/cmd/testserver/testserver.go

```diff
@@ -84,6 +84,7 @@ func (c *Config) Complete() (RunnableTestServer, error) {
 				MaxLookupResourcesLimit:         c.MaxLookupResourcesLimit,
 				MaxBulkExportRelationshipsLimit: c.MaxBulkExportRelationshipsLimit,
 				DispatchChunkSize:               defaultMaxChunkSize,
+				ExpiringRelationshipsEnabled:    true,
```

Comment:
Same here - should this reference the config?

Reply:
No; I think its fine to enable by default in the test server